exercises/concept/paulas-palindromes/.docs/hints.md

```diff
@@ -0,0 +1,40 @@
+# Hints
+
+## General
+
+- Check out the [Elm Guide][elm-guide], which has a nice worked example that is similar.
+
+## 1. Define the Model and Msg types for the application
+
+- The `Model` contains the application's state - all the data that the application needs.
+- This application just needs a string to store the text in the Text Box (the tests require it to be called `content`).
+- It can be any type, but in any useful application it is usually a [record][record].
```

Comment:
Let's move this suggestion to be in second place, because the hint above mentions the field name
```suggestion
- It should be a record][record].
```